kf_lib_data_ingest/etl/load/load.py

```diff
@@ -347,12 +352,19 @@ def _load_entity(self, entity_type, endpoint, entity_id, body, links):
                 self.resume_from = None
 
         if self.dry_run:
-            # Fake sending with fake foreign keys
```

Comment:
I changed this because I didn't think prefixing each foreign key's source value with `DRY` was that useful. I think we should either do what I'm doing now (primary/foreign keys show up like `source: <source id val> --> target: <target id val>` or leave the body of the request alone entirely in dry-run. I prefer option 1